src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -1144,7 +1145,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!-- Remove projects that do not support restore. -->
     <!-- With SkipNonexistentTargets -->
-    <MsBuild
+    <MSBuild
```

Comment:
Why this change?

Reply:
Response: https://github.com/NuGet/NuGet.Client/pull/3538#issuecomment-668995259